src/unity/python/turicreate/visualization/plot.py

```diff
@@ -5,9 +5,20 @@
 import json as _json
 import os as _os
 from tempfile import mkstemp as _mkstemp
+from subprocess import Popen as _Popen 
+from subprocess import PIPE as _PIPE
 
 _target = 'auto'
 
+NODE_NOT_FOUND_ERROR_CODE = 127
```

Comment:
Unless you want to publicly export this from this file, you should start the name with an underscore.

Reply:
Oh yeah good catch, thanks!!